guides/Tong/README.md

```diff
@@ -0,0 +1,76 @@
+## Written by Sharon Tong
+_________________________
+
+### Contact
+- Slack: @Sharon on wcscusf.slack.com
+- LinkedIn: linkedin.com/in/sharon-tong
+- Email: sharont1@mail.usf.edu
+### Table of Contents
+1. [Prerequisites](#id-link-to-section)
+2. [Summary](#id-link-to-section)
+3. [Installing PFsense](#id-link-to-section)
+4. [Configuring PFsense](#id-link-to-section)
+5. [Troubleshooting](#id-link-to-section)
+
+## Prerequisites <a id="id-link-to-section"></a>
+1. Have your [virtual environment](https://www.virtualbox.org/wiki/Downloads) configured.
```

Comment:
Once this is merged along with Nestor's guides make another pull request to add the appropriate links. For now this is good, just a future note. Along with line 18.